app/serializers/project_serializer.rb

```diff
@@ -0,0 +1,4 @@
+class ProjectSerializer < ActiveModel::Serializer
+  attributes :id, :name, :slug, :owner_id, :created_at, :updated_at, :deleted_at, :team_id
```

Comment:
Yeah potentially. I think if we are to return extra resources we should take URL params to do so like:

`get /api/v1/projects/:slug?includes=comments`